app/controllers/profiles/personal_access_tokens_controller.rb

```diff
@@ -32,10 +32,13 @@ def create
 
     def revoke
       @personal_access_token = current_user.personal_access_tokens.find(params[:id])
-      @personal_access_token.revoke!
+      if @personal_access_token.revoke!
 
-      respond_to do |format|
-        format.turbo_stream
+        respond_to do |format|
+          format.turbo_stream
+        end
+      else
+        format.turbo_stream { render 'shared/error/not_found' }
```

Comment:
Instead of showing this partial, you should render an error message that would show up in the table. If I manually change the `Revoke` button to a non existent id, I get the following.
![image](https://github.com/phac-nml/irida-next/assets/492127/bb8e5b65-597b-4746-8b4d-50bd4ec30501)

Reply:
Actually can you update this to use the same styling Josh added in login view for showing errors.

Reply:
Updated to use the viral_alert component in 5b0317446b5f8e0cd4ae4a600a9a28e48d3a74ce.

Reply:
![image](https://github.com/phac-nml/irida-next/assets/25466211/c7e635f6-dca4-4d4f-a946-6b97377c21c5)